test/e2e/scheduling/priorities.go

```diff
@@ -303,19 +303,40 @@ var _ = SIGDescribe("SchedulerPriorities [Serial]", func() {
 		// Apply 10 taints to first node
 		nodeName := nodeList.Items[0].Name
 
-		ginkgo.By("Trying to apply 10 (tolerable) taints on the first node.")
```

Comment:
why was this removed?

Reply:
It was moved down to where those taints are actually applied (L330)

Reply:
sg